apps/emqx_message_validation/src/emqx_message_validation.erl

```diff
@@ -69,10 +72,10 @@ remove_handler() ->
     ok.
 
 load() ->
-    lists:foreach(fun insert/1, emqx:get_config(?VALIDATIONS_CONF_PATH, [])).
+    lists:foreach(fun insert_local/1, emqx:get_config(?VALIDATIONS_CONF_PATH, [])).
```

Comment:
load/0 should only do whatever is to be done in the config handler.
but not to trigger a config update.

Reply:
:heavy_check_mark: